dashboard/config/locales/en.yml

```diff
@@ -156,7 +158,7 @@ en:
     student_consent: "I confirm that I have my parent or legal guardian's permission to use the Code.org services."
     agree_us_website: "I agree to using a website based in the United States."
     my_data_to_us: "Data from my use of this site may be sent to and stored or processed in the United States."
-    student_terms: "Email addresses are not stored in a form that allows us to contact students. Students will never receive emails from Code.org except for password recovery. See our <a href='http://code.org/privacy'>privacy policy</a> for more information."
+    student_terms: "\\* Email addresses are not stored in a form that allows us to contact students. Students will never receive emails from Code.org except for password recovery. See our [privacy policy](%{url}) for more information."
```

Comment:
we actually want to create a new string for this; yes, we'll lose translations, but the old translations include HTML which will get stripped out